src/Reddit/Reddit.ts

```diff
@@ -236,12 +543,13 @@ export default <IPluginBase & IPlugin>{
         match: ["collapse #", "# collapse"],
         normal: false,
         pageFn: (transcript, index: number) => {
-          const el = <HTMLElement>(
-            document.querySelector(
-              thingAtIndex(index) + " .expando-button:not(.collapsed)"
-            )
+          const {comments, special} = reddit.old;
```

Comment:
it always uses reddit.old?

Reply:
Yes, because in new reddit there is no expandable elements